e2e/paths/deleteCase_test.js

```diff
@@ -0,0 +1,17 @@
+const config = require('../config.js');
+
+Feature('Submit Case').retry(2);
+Before((I, caseViewPage) => {
+  I.logInAndCreateCase(config.swanseaLocalAuthorityEmailUserOne, config.localAuthorityPassword);
+  I.selectOption(caseViewPage.actionsDropdown, config.applicationActions.deleteCase);
```

Comment:
Please move it to scenario. To me selecting option is an action that is essential for the test and as such it should not be defined in precondition / prepare stage.

Reply:
I have refactored, to follow other tests